### PR TITLE
HTML (with CSS & JS) project support

### LINKED ISSUE #45: Vanilla HTML, CSS, JS Project Support
With appropriate folders too

### PR DESCRIPTION
**Pull request by jarodburchill**
Closes #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ Blank repository with a README:
 ```
 blank
 ```
+HTML boilerplate that comes CSS and JS ready:
```
Comment (Rexogamer) [inline README.md]: I don’t quite know why, but this just feels grammatically wrong. Maybe rewrite it as “HTML boilerplate complete with CSS and JS”?

Comment (jarodburchill) [reply README.md]: Oh, I forgot the word 'with' after 'that comes'... but yours sounds better!

Comment (jarodburchill) [reply README.md]: Yours sounds good, change it to that!

**Review by Rexogamer — APPROVED** [review]
Looks good!

**Pull request merged**